src/pages/hackathon/redacted.tsx

```
@@ -0,0 +1,246 @@
+            </div>
+          </div>
+        </div>
+        <div>
+          <FAQs />
+        </div>
+      </div>
+    </Default>
+  );
+}
+
+export const getServerSideProps: GetServerSideProps = async ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)